packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -46,7 +46,7 @@ export interface IComponentPageProps {
   /** Native props root element */
   nativePropsElement?: string | string[] | undefined;
   /** Includes the feedback section **/
-  isFeedbackVisible?: boolean;
+  isFeedbackVisible?: JSX.Element;
```

Comment:
if this prop is meant to pass in the feedback, we need to change the prop name.

Reply:
In the individual component doc pages this is still a boolean and only accepts true or false based on whether they want the feedback to show up or not. That is why I left this name since it is a JSX element here but a boolean in DocPage

Reply:
We always want to make sure our interfaces are consistent. If the interface is used in multiple places, it should be the same type.

So if you need a prop to pass in some JSX.Element, lets add that prop.

Reply:
made the required changes